libnmstate/nm/ovs.py

```diff
@@ -50,6 +53,20 @@ def has_ovs_capability():
     return nmclient.NM.Capability.OVS in nm_client.get_capabilities()
 
 
+def is_ovs_running():
```

Comment:
This function does not related to NetworkManager, please place it in `appliers/ovs.py` initially(I will find better place for it later).

Reply:
Done, thanks! Maybe we could create something like `libnmstate/utils/ovs.py` and do so for other interface types.  What do you think? I move it to `appliers/ovs_bridge.py`. Thanks!